borsh-schema-derive-internal/src/enum_schema.rs

```diff
@@ -362,6 +362,59 @@ mod tests {
         })
         .unwrap();
 
+        let actual = process_enum(&item_struct, Ident::new("borsh", Span::call_site()));
+
+        let err = match actual {
+            Ok(..) => unreachable!("expecting error here"),
+            Err(err) => err,
+        };
+        insta::assert_debug_snapshot!(err);
```

Comment:
I feel unwrapping the error would be enough and easier to read the code:

```suggestion
        insta::assert_debug_snapshot!(actual.unwrap_err());
```

Reply:
There was something about the specific type not implementing `Debug` (and consequently `.unwrap_err()` not being available)

Reply:
Yes, at this spot and in `struct_schema.rs` `T: Debug` and `unwrap_err()` are available. 
It is not available in `attribute_helpers/field.rs` tests.